src/components/dashboard/SettingRow.tsx

```
@@ -0,0 +1,129 @@
+
+      <td className="p-4">
+        {setting.isEditable && (
+          <div className="flex items-center space-x-2">
+            {editMode ? (
+              <>
+                <button
+                  onClick={handleSave}
+                  disabled={loading}
+                  className="p-1.5 text-white bg-green-600 rounded-md hover:bg-green-700 focus:outline-none focus:ring-2 focus:ring-green-500 focus:ring-offset-2 disabled:opacity-50 disabled:cursor-not-allowed"
+                >
+                  <Save className="w-4 h-4" />
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)